tests/acceptance/features/apiSpaces/publicLink.feature

```diff
@@ -0,0 +1,35 @@
+@api @skipOnOcV10
+Feature: As a viewer (public)
+  I should not be able to create/delete/upload/change resources
```

Comment:
will this feature be used only for Viewer role? Also, you can add better feature description :smile: